apps/meteor/app/apps/server/bridges/rooms.ts

```diff
@@ -102,6 +102,55 @@ export class AppRoomBridge extends RoomBridge {
 		return this.orch.getConverters()?.get('users').convertById(room.u._id);
 	}
 
+	protected async getMessages(
+		roomId: string,
+		options: {
+			limit: number;
+			skip?: number;
+			sort?: Record<string, 1 | -1>;
+		},
+		appId: string,
+	): Promise<IMessageRaw[]> {
+		this.orch.debugLog(`The App ${appId} is getting the messages of the room: "${roomId}"`);
```

Comment:
This one should login the pagination options passed as well, so we can track what the app was doing

Reply:
Thank you! Nice idea.